### ISSUE TITLE
NVDA Losing Linebreaks in PDF Files

### ISSUE THREAD

**Issue by dan1982code** [nvaccess/nvda#7337]
Open a PDF file with Acrobat Reader, such as the one attached here. Whether in "infer reading order from document" or "left-to-right" reading mode, NVDA squishes lines together that should be on separate lines. Saving the PDF file as text demonstrates that the linebreaks are preserved in the text stream. Other screen-readers do honour the linebreaks, which makes PDF files quite a bit easier to read.
[test.pdf](https://github.com/nvaccess/nvda/files/1110516/test.pdf)

**Comment by dan1982code** [comment]
Sorry, duplicate.
